python-sdk/tutorials/automl-with-azureml/forecasting-many-models/auto-ml-forecasting-many-models.ipynb

```diff
@@ -242,6 +242,35 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 2.4 Configure data with OutputFileDatasetConfig objects (Optional)\n",
+    "Intermediate data (or output of a step) is represented by an OutputFileDatasetConfig object. output_data is produced as the output of a step. Optionally, this data can be registered as a dataset by calling register_on_complete. If you create an OutputFileDatasetConfig in one step and use it as an input to another step, that data dependency between steps creates an implicit execution order in the pipeline.\n",
+    "\n",
+    "OutputFileDatasetConfig objects return a directory, and by default writes output to the default datastore of the workspace.\n",
+    "\n",
+    "Since instance creation for class OutputTabularDatasetConfig is not allowed, we first create an instance of class OutputFileDatasetConfig. Then we use method read_parquet_files to read the parquet file into OutputTabularDatasetConfig."
```

Comment:
Please change to "Then we use the read_parquet_files method."

Reply:
done.